### ISSUE TITLE
stage: don't write default wdir

### ISSUE THREAD

**Issue by efiop** [iterative/dvc#2443]
https://github.com/iterative/dvc/blob/0.57.0/dvc/stage.py#L632

`.` doesn't evaluate to False and so not skipped.

**Comment by pared** [comment]
What is the reasoning for not writing `wdir`? We actually have [test](https://github.com/iterative/dvc/blob/41351a2df2f36ed04846d5e3d976d6d99b71a151/tests/func/test_run.py#L563) checking that wdir `.` is written.

**Comment by efiop** [comment]
@pared just not printing default values, to make it look nicer. If wdir is not specified we default it to `'.'` anyway https://github.com/iterative/dvc/blob/0.58.0/dvc/stage.py#L453 , so it seems like we could simply remove it. Good point about that test, it was a part of https://github.com/iterative/dvc/commit/e26db30f6039a82f00a54475a0dc9a34b7861937 , and might've been just a sanity check, but we need to take a closer look to confirm that.

**Comment by pared** [comment]
@shcheklein, do you remember if test `test_default_wdir_is_written` was just sanity check?
I think the most important is that default wdir is ignored in checksum, right?

**Comment by shcheklein** [comment]
@pared Yes, I think it was a sanity check. And yes, default wdir ignored in checksum is an important one. I would keep the sanity check anyway, just change it appropriately.

**Comment by pared** [comment]
@efiop 
regretfully, https://github.com/iterative/dvc/blob/0.58.0/dvc/stage.py#L453 can happen only if `cwd`, which is obsolete is specified, so removing assignment frome there is not enough.

I prepared solution for current issue, but I also believe that we should create new issue, which would take care of using `wdir` inside `Stage.create`. Currently there are 2 places ((here)[https://github.com/iterative/dvc/blob/79cd3d670fa3421d49361d9fae114199605234ad/dvc/stage.py#L437] and (here)[https://github.com/iterative/dvc/blob/79cd3d670fa3421d49361d9fae114199605234ad/dvc/stage.py#L476]) where we get its value from kwargs, which makes logic invloved with `wdir` not clear.
